Taplytics Web Integration

jsalaber opened the pull request.

rmi22186 commented on TaplyticsKit.js:
```diff
@@ -1 +1,301 @@
+            if (isInitialized) {
+                try {
+                    Taplytics.identify({
+                        [key]: value
```
Again, ES5 only for now :(

so you'd need to do a 

```
var attributes = {}, 
attributes[key] = value
Taplytics.identify(attributes)
```
here

rmi22186 commented on TaplyticsKit.js:
```diff
@@ -1 +1,301 @@
+                src = src + '&cookieDomain=' + cookieDomain;
+            }
+            
+            const user_attributes = encodeURIComponent(JSON.stringify(initUserAttributes));
```
Right now we don't run our integrations through any compilers to be compatible with all legacy browsers, so it is safest to write everything in es5

var here

rmi22186 commented on TaplyticsKit.js:
```diff
@@ -1 +1,301 @@
+                var id = identity.Identity;
+                switch (type) {
+                    case 1:
+                        user_attributes["user_id"] = id;
```
user attributes is a string right now, from line 153, so you'd probably want to stringify it below right before you add it to the source rather on line 153

jsalaber commented:
@rmi22186 I addressed your comments. Right now, our SDK is pretty simple so the ones that I implemented are pretty much the only events we support as of now.

rmi22186 commented on TaplyticsKit.js:
```diff
@@ -140,20 +140,21 @@
 
             if (cookieDomain) {
-                src = src + '&cookieDomain=' + cookieDomain;
+                src = src + (query ? '&' : '') + 'cookieDomain=' + cookieDomain;
```
since `src` is just the above URL for now, i assume that it shouldn't be mutated until the end once we know what the query parameters all are. so we should probably just keep building the query parameters out without requiring the src:
 
`query = query + ( query ?  '&' : '') + 'cookieDomian=' + cookieDomian`

rmi22186 replied:
whoops.  i mean just 1 comment

jsalaber replied:
oops yeah that should be query
